neovim/__init__.py

```diff
@@ -118,7 +118,7 @@ def setup_logging(name):
     if 'NVIM_PYTHON_LOG_FILE' in os.environ:
         prefix = os.environ['NVIM_PYTHON_LOG_FILE'].strip()
         major_version = sys.version_info[0]
-        logfile = '{0}_py{1}_{2}'.format(prefix, major_version, name)
+        logfile = '{}_py{}_{}'.format(prefix, major_version, name)
         handler = logging.FileHandler(logfile, 'w')
```

Comment:
I didn't mean I didn't _understand_ what it it means, just that it makes no _sense_ . If the reader is able to count the arguments on the right side, then the reader should be capable to do that on the left side as well. If a case is so complicated that cues are needed, then those should be on _both_ sides, using named arguments.